### ISSUE TITLE
Can't print vector.outer() products

### ISSUE THREAD

**Issue by ajlawren** [sympy/sympy#26279]
Printing anything involving vector.outer() products fails. 
```from sympy.vector import CoordSys3D
N = CoordSys3D('N')
Av = 1*N.i
Av.outer(Av)
```
Results in output as raw text, highlighted yellow in red font: 
```\displaystyle \left(\mathbf{\hat{i}_{N}}{\middle|}\mathbf{\hat{i}_{N}}\right)```

All other printed output succeeds. Tested in latest JupyterLab and Sympy Live

**Comment by HimaTeju** [comment]
I tried the same code in Google Collab:
'''from sympy.vector import CoordSys3D
N = CoordSys3D('N')
Av = 1*N.i
Av.outer(Av)'''
'''(N.i|N.i)'''

**Comment by HimaTeju** [comment]
I tried the same code in Google Collab:
```from sympy.vector import CoordSys3D
N = CoordSys3D('N')
Av = 1*N.i
Av.outer(Av)```
```(N.i|N.i)```

**Comment by mohamedrezk122** [comment]
Here is your example 
```python
>>> from sympy.vector import CoordSys3D
>>> N = CoordSys3D('N')
>>> Av = 1*N.i
>>> Av.outer(Av)
```
```
\displaystyle \left(\mathbf{\hat{i}_{N}}{\middle|}\mathbf{\hat{i}_{N}}\right)
```
**Instead, use** `init_printing()` function
```python
>>> from sympy.vector import CoordSys3D
>>> from sympy import init_printing
>>> init_printing(use_latex="math_jax")
>>> N = CoordSys3D('N')
>>> Av = 1*N.i
>>> Av.outer(Av)
```
```
(i_N|i_N)
```

**Comment by ajlawren** [comment]
I now see that this fails when `init_printing(use_latex=True)`, but `init_printing(use_latex=False)` still works.

**Comment by Rohanberiwal** [comment]
Hi,

I've made some progress on the issue. I used the pprint function from SymPy to print the result of the dyadic representation of the outer product of a vector along the x-axis with itself. The vector has a unit magnitude. I've attached the code below along with the output. I tested it on Jupyter Notebook.The output is (i_N|i_N) .
Code -> 

from sympy import pprint
from sympy.vector import CoordSys3D

N = CoordSys3D('N')
Av = 1*N.i
outer_product = Av.outer(Av)

pprint(outer_product, use_unicode=True)

 Please let me know if the apporach is correct  .

**Comment by ajlawren** [comment]
Yes, it is correct, but that disables pretty print in Mathjax.

**Comment by Rohanberiwal** [comment]
Thanks for your response. Is it acceptable to proceed with the current approach, even though it disables pretty print in Mathjax? Alternatively, should I explore alternative methods to maintain pretty printing in mathjax while addressing the issue?
Looking forward to your guidance.

**Comment by Rohanberiwal** [comment]
Oh ! Got that , i was actually running the whole code and the print in the same cell , that's why i was not able to figure out the error that we were supposed to eliminate  . I will get back .. Thanks !!

**Comment by donnaaboise** [comment]
I am running into the same issue in a Jupyter notebook.   One issue seems to be with the latex formatting. Specifically, the braces around the `\middle|` causes the problem .  

Here is a complete code and a possible fix : 

```
from IPython.display import Markdown
import sympy as sp
from sympy.vector import CoordSys3D
R = CoordSys3D('R')

dyad = R.i.outer(R.j)
display(dyad)

print(sp.latex(dyad))

# remove braces around `\middle|`. 
latex_str_good = r"$\left(\mathbf{\hat{i}_{R}}\middle|\mathbf{\hat{j}_{R}}\right)$"
display(Markdown(latex_str_good))
```

The original string prints as 

```
\left(\mathbf{\hat{i}_{R}}{\middle|}\mathbf{\hat{j}_{R}}\right)
```
The string, with the braces removed from the `\middle|` command,  displays as expected. 

@Rohanberiwal : The fix you suggested works with the `print` statement.  But if you replace `print` with `display`, you'll see the error posted by the @ajlawren.  And the latex string still has the braces around `\middle|`.  The issue has nothing to do with whether the code is printed/displayed in the same cell or not.

BTW, is it possible to optionally print the dyad symbol print as `$\otimes$` rather than `$|$`?
